### ISSUE TITLE
Broken release edit page?

### ISSUE THREAD

**Issue by adrian-evo** [sw360/sw360portal#526]
Deployed the latest branch master (2d10feb), and when trying to edit some of the release fields, it is not working. For example Vendor dialog is not opened and the same with moderators dialog, Programming languages and operating systems autofill also not working.

The error from the log is:
2017-08-01 12:19:12,086 ERROR FossologyAwarePortlet:178 - Could not get release from request
2017-08-01 12:19:12,088 ERROR FossologyAwarePortlet:146 - no response from backend!

**Comment by maximilianhuber** [comment]
@lepokle is this maybe related to https://github.com/sw360/sw360portal/pull/506 (together with caching, unclean deployment, ...)

**Comment by lepokle** [comment]
This is possible. The release edit page was changed during #506 but all tests did work.
Please be sure to have a clean deployment and to throw away any browser cache.
If this does not help, please provide console output of the browser (F12).

**Comment by adrian-evo** [comment]
Tried with clean deployment and cache. When editing a release, the following is displayed.
![release](https://user-images.githubusercontent.com/19324942/28861178-69a1c2a0-7768-11e7-94ec-9478e55659c4.PNG)

**Comment by lepokle** [comment]
Sorry, I'm still not able to reproduce this issue. May you please tell me your browser version? Thx.

**Comment by lepokle** [comment]
Sorry, I'm still not able to reproduce this issue. May you please tell me your browser version? Thx.

**Comment by adrian-evo** [comment]
This is from Firefox 52.2. ESR. The next images are from Chrome v59 and IE11. On IE11 only I also get exception when clicking on Programming Languages field. But if these are not relevant and you cannot reproduce the issue, maybe I should try with another branch to see if something was wrong during my deployment.
![chrome59](https://user-images.githubusercontent.com/19324942/28970174-3acbd23a-7930-11e7-8bc7-bd37199d9e28.PNG)
![explorer11](https://user-images.githubusercontent.com/19324942/28970185-46465f7c-7930-11e7-8380-66f913b8c7a1.PNG)

**Comment by ChristophNiehoff** [comment]
Also the "Click to add Releases" button on the release edit page appears to be broken. (This was confirmed by @stevinus1 ).

**Comment by lepokle** [comment]
@adrian-evo Hi Adrian,
it seems like your proxy settings are outdated causing a forbidden error:
![image](https://user-images.githubusercontent.com/499823/29060748-5fba4ad4-7c1b-11e7-8295-2fae55ec72c8.png)
We've updated the rules in the sw360-chores project. However I've forgot to update the rules in vagrant as well. I'll comment here as soon as this rules has been updated as well.

If you use a custom nginx proxy be sure to change the settings as followed.
Old settings:
```
location ~* /(manager|attachments|bdpimport|components|cvesearch|fossology|layouttpl|licenseinfo|licenses|moderation|projects|schedule|search|users|vendors|vulnerabilities)/ {
        deny all;
}
```

New settings:
```
location ~* ^/(manager|attachments|bdpimport|components|cvesearch|fossology|layouttpl|licenseinfo|licenses|moderation|projects|schedule|search|users|vendors|vulnerabilities)/ {
        deny all;
}
```
Note the `^` to match the beginning of the URL

The old settings did not allow a path fragments like 'manager' or 'component' anywhere in the URL while the new settings only restrict these path fragments to the beginning of the URL.
